### PR TITLE
Merge develop to master, no version bump needed

### PR DESCRIPTION
**Pull request by vsudilov**

### CODE REVIEW

**Comment by vsudilov** [comment]
@jonnybazookatone feel free to be a reviewer too

**Comment by vsudilov** [comment]
Welp, any ideas why travis isn't reporting coverage to coveralls.io ? I notice that in 2 of the 4 (py3.5, 3.6) builds I see a terminal coverage report, and for the others I don't. In my local python 2.7.12 virtualenv, I do see a terminal report (and .coverage file) being generated after `python setup.py test`

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        :return: Empty response with status 200, 429 or 500
+        """
+        # TODO: implement any other checking logic.
+        return '', 200
+    
+    
+    # Consul
+    # This extension should be the first one if enabled:
+    consul = Consul(app=app)
+    # Fetch the conviguration:
```
Comment (jonnybazookatone) [inline README.rst:53]: typo

**Comment by sobolevn** [comment]
@vsudilov it shows coverage in travis: https://travis-ci.org/vsudilov/flask-consulate/jobs/221525908#L455

**Comment by sobolevn** [comment]
@vsudilov maybe because of this warning: 

> /home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/pkg_resources/__init__.py:2510: PEP440Warning: 'pytest (cov-1.7.0)' is being parsed as a legacy, non PEP 440, version. You may find odd behavior and sort order. In particular it will be sorted as less than 0.0. It is recommend to migrate to PEP 440 compatible versions.

Link: https://travis-ci.org/vsudilov/flask-consulate/jobs/221525909#L477 and https://travis-ci.org/vsudilov/flask-consulate/jobs/221525906#L497

This warning persists in `2.7` and in `3.4`.

**Comment by jonnybazookatone** [comment]
https://travis-ci.org/vsudilov/flask-consulate/jobs/221525907#L492

```bash
$ coveralls

/home/travis/.travis/job_stages: line 53: coveralls: command not found
```

**Comment by jonnybazookatone** [comment]
Possibly a stupid question. Does the tox file need to be configured for coveralls as well? (https://github.com/coveralls-clients/coveralls-python#usage-tox--v20)

File: README.rst (hunk 1 above)
Comment (vsudilov) [reply README.rst:53]: Yeah let's not worry about this during a release merge -- but good catch

**Comment by vsudilov** [comment]
@jonnybazookatone we aren't using tox, and coverage/coveralls isn't dependent on tox. pytest-cov should be providing the coverage directly (and is for certain versions of python...)

@sobolevn I did see that warning, but assumed it was fine since it does find the latest version:
```
Best match: pytest-cov 2.4.0
Downloading https://pypi.python.org/packages/00/c0/2bfd1fcdb9d407b8ac8185b1cb5ff458105c6b207a9a7f0e13032de9828f/pytest-cov-2.4.0.tar.gz#md5=2fda09677d232acc99ec1b3c5831e33f
Processing pytest-cov-2.4.0.tar.gz
```

Will see if I can repro locally with the various python versions and poke around more.

**Pull request closed without merging**